dlt/common/pendulum.py

```diff
@@ -2,7 +2,7 @@
 import pendulum  # noqa: I251
 
 # force UTC as the local timezone to prevent local dates to be written to dbs
-pendulum.set_local_timezone(pendulum.timezone('UTC'))
+pendulum.set_local_timezone(pendulum.timezone('UTC')) # type: ignore
```

Comment:
This errors with `Unused "type: ignore" comment  [unused-ignore]` in CI
https://github.com/dlt-hub/dlt/actions/runs/6786606444/job/18447614187?pr=724

Reply:
Weird that running lint on my machine works fine. Removing this ignore label errors.

```bash
make lint
./check-package.sh
poetry run mypy --config-file mypy.ini dlt tests
dlt/common/pendulum.py:5: error: Module not callable  [operator]
Found 1 error in 1 file (checked 474 source files)
make: *** [lint] Error 1
```

Reply:
I committed the change nevertheless.

Reply:
Passed.